Source/Libraries/GSF.Web/Model/ModelController.cs

```diff
@@ -823,7 +824,7 @@ SELECT TOP 0 * INTO #Tbl FROM {tblSelect}
                                 FROM tempdb.sys.columns WHERE  object_id = Object_id('tempdb..#Tbl') AND name NOT LIKE 'AFV%';";
                     sqlNoPivot = string.Join(",", connection.RetrieveData(sqlNoPivot).Select().Select(r => r[0].ToString()));
                     sql = $@"
-                        SELECT {limit} {sqlNoPivot} FROM {tblSelect}
+                        SELECT {limit} {((postData?.ReturnPivotCols ?? false) ? "*" : sqlNoPivot)} FROM {tblSelect}
```

Comment:
That wasn't really the intention here.

Make it a `protected bool ReturnPivotCols  => false` on the Controller.
No need to change `PostData`